### ISSUE TITLE
Staging Review finding: Add the privacy agreement and penalty notice to review page

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98990]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Accredited Representation Management
**Product:** Appoint a Representative MVP
**Feature:** Appoint a Representative + Representative Status Widget

## Findings details
**VA.gov Experience Standard - issue:** [User encounters consistent, clear design components and patterns.](https://design.va.gov/about/experience-standards/#user-encounters-consistent-clear-design-components)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @allison0034 (Design)

### Description

[Penalty notice](https://design.va.gov/components/form/penalty-notice) is required for all forms. [Privacy policy](https://design.va.gov/components/form/privacy-agreement) is required for all forms with PII. Your review page does not have either.

<details><summary>Link, screenshot or steps to recreate</summary>

![screen shot](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/71535-allison-5.png?raw=true)

</details>

## Recommended action

Add both to your [review page](https://design.va.gov/templates/forms/review). 

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** va-privacy-agreement
- **Design System Pattern or Template:** [Forms - Review](https://design.va.gov/templates/forms/review)
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by oddball-lindsay** [comment]
Assigning @holdenhinkle to pick up the development, but I think we might need some design help first @mtri as I have some open questions:
1. How do we add in the [Penalty Notice "note"](https://design.va.gov/components/form/penalty-notice) when we already have a note? 
2. How do we add in the additional [Privacy Agreement checkbox](https://design.va.gov/components/form/privacy-agreement), given we already have two checkboxes on the page?

**Comment by mtri** [comment]
I processed these questions whipping up some examples:

[Option 1: Order UI and content would be in based on the my interpretation of the VADS documentation](https://www.figma.com/design/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?node-id=13450-222679&node-type=frame&t=UMctbmjyTpNh8CjX-11)

[Option 2: Order I would put things in myself](https://www.figma.com/design/bzbwObT9hiItve0q3cQX9c/Find-and-Appoint-a-Representative?node-id=13450-221449&node-type=frame&t=UMctbmjyTpNh8CjX-11)

- I would recommend the order in Option 2 just because it reads in a more natural way.
- If Option 1 is a must and we can’t deviate, we should ask CAIA about placing two notes next to each other. They might want to consolidate somehow.

**Comment by oddball-lindsay** [comment]
Thank you so much @mtri for the quick thinking!

I also appreciate the Option 2 where all the actionable items are grouped together. @allison0034 any thoughts from you before we head down developing the Option 2 pathway?

**Comment by allison0034** [comment]
@mtri @oddball-lindsay I agree with Michael's suggestion as long as CAIA is ok with it. Tagging @briandeconinck to see if he has any concerns.

Michael, I would change your Figma button to "Continue" since that is the way it is in Staging. Users are not actually "submitting" anything. Just want to make sure CAIA is clear on what is happening. If you get any push back you could make your "note" an alert at the top of the page if you feel it is that important that it needs to stand out. We do have alert fatigue on VA so if your note tested fine then I would leave it if possible.

Thank you both so much for the great work!

**Comment by holdenhinkle** [comment]
I'll wait until @briandeconinck responds before starting this.

**Comment by briandeconinck** [comment]
No concerns, I think you're good to proceed.

**Comment by holdenhinkle** [comment]
Thank you!

**Comment by holdenhinkle** [comment]
I'm going to implement Michael's Option 2

**Comment by holdenhinkle** [comment]
@mtri Ready for review!

**Comment by holdenhinkle** [comment]
@mtri an important note from the PR description:

The link to the privacy policy in this PR does not match the design -- the design has an icon that indicates the link opens in a new page, while the link in this PR does not display the icon but instead has text that says "[(opens in a new tab)opens in a new tab](http://localhost:3001/privacy-policy/)". This text, as well as all elements and design of this privacy policy checkbox, come from the VA web component.

**Comment by mtri** [comment]
@holdenhinkle Looks good! Tested on Desktop and Mobile. 

Is it possible to update the bottom note content to “**Note:** You can't edit the information for your form after this step.”? This was a change living in a Figma comment that fell through the cracks.

**Comment by holdenhinkle** [comment]
PR to update the note - https://github.com/department-of-veterans-affairs/vets-website/pull/33733

**Comment by holdenhinkle** [comment]
@mtri

**Comment by oddball-lindsay** [comment]
Looks good!
